elide-graphql/src/test/java/example/Book.java

```diff
@@ -22,6 +22,11 @@
 import com.yahoo.elide.annotation.SharePermission;
 import com.yahoo.elide.security.RequestScope;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.Builder;
```

Comment:
Is this unused?

Reply:
This is used in a javadoc reference.

```
/**
 * Model for book.
 * <p>
 * <b>CAUTION: DO NOT DECORATE IT WITH {@link Builder}, which hides its no-args constructor. This will result in
 * runtime error at places such as {@code entityClass.newInstance();}</b>
 */
```

Reply:
I meant the Builder import.